rest/views.py

```diff
@@ -48,6 +49,39 @@
 from django.http.response import Http404
 
 
+def get_urls(raw_urls, list_urls, urlbase=''):
+    """
+    Recursively builds a list of all the urls in the current project and
+    the name of their associated view
+    """
+    list_urls = list_urls or []
+    from operator import itemgetter
+    for entry in raw_urls:
+        full_url = (urlbase + entry.regex.pattern).replace('^', '')
```

Comment:
If we used [DefaultRouter](http://www.django-rest-framework.org/api-guide/routers/#defaultrouter) it will automatically create these URLs for us, however it doesn't know about nested resources so we would have a weird combination of autogenerated URL regexes and manually specified ones